crates/ingress-kafka/src/consumer_task.rs

```diff
@@ -418,13 +412,11 @@ impl ConsumerContext for RebalanceContext {
     }
 }
 
-struct AbortOnDrop(TaskCenter, TaskId);
+struct AbortOnDrop(TaskHandle<()>);
```

Comment:
Optional: This is a handy type if you want to move to task_center's so others can also use it.